src/chains/ethereum/ethereum/src/blockchain.ts

```diff
@@ -61,7 +61,7 @@ import {
 } from "@ethereumjs/vm/dist/state/index";
 import { GanacheTrie } from "./helpers/trie";
 import { ForkTrie } from "./forking/trie";
-import { LevelUp } from "levelup";
+import type { LevelUp } from "levelup";
```

Comment:
The types got weird at one point, so I was attempting to figure it out and one of the steps was to ensure that we were using only the type, and not the runtime. Since many of these imports really are type-only imports I've left these changes in.